### ISSUE TITLE
A working example for 4.5/4.6.

### ISSUE THREAD

**Issue by cduhard** [lvermeulen/Nanophone#12]
How would I use this in 4.5-4.6 environment? I understand wanting to work with the latest, but some working samples in 4.5-4.6 would be great. I can't imagine many people have the option to be deploying dotnetcore apps in the near future. 

The sample for Nancy isn't actually Nancy.

**Comment by lvermeulen** [comment]
Well you install the package(s) from NuGet, just make sure you're using the latest prerelease.

> Install-Package Nanophone.RegistryTenant.Nancy -Pre

The sample is definitely Nancy, it's practically lifted from Nancy's own sample. :-) I used to have .net45 samples but I removed them. I'll see about adding them back sometime this week.

**Comment by cduhard** [comment]
The Nancy looks like below. It references WebApiRegistryTenant. I was referring to that when I said it doesn't use Nancy. 

```
public static void Main()
        {
            const bool USING_FABIO = false;

            var log = LogManager.GetCurrentClassLogger();
            log.Debug($"Starting {typeof(Program).Namespace}");

            const int PORT = 9060;
            string url = $"http://localhost:{PORT}/";
            var consulRegistryHost = new ConsulRegistryHost();
            var serviceRegistry = new ServiceRegistry(consulRegistryHost);

            if (USING_FABIO)
            {
                var fabioHandler = new FabioAdapter(new Uri("http://localhost:9999"));
                serviceRegistry.ResolveServiceInstancesWith(fabioHandler);
            }

            serviceRegistry.AddTenant(new WebApiRegistryTenant(new Uri(url)),
                "orders", "1.3", keyValuePairs: new[] { new KeyValuePair<string, string>("urlprefix-", "/orders") })
                .Wait();

            var host = new WebHostBuilder()
                .UseKestrel()
                .UseUrls($"http://*:{PORT}")
                .UseContentRoot(Directory.GetCurrentDirectory())
                .UseStartup<Startup>()
                .Build();

            host.Run();
}
```

I see you've added a 4.6 example. I'll give nanophone another try. It may be hidden to you, but trying to get these hybrid dotnetcore/4.6 solutions working for those of us not using dotnetcore can be painful.

**Comment by lvermeulen** [comment]
That's ... embarrassing. :-/ Fixed and pushed, thanks for pointing that out.
The 4.6 sample is not finished, I only just started to add it but haven't tested it yet. 

I guess the hybrid problem may indeed be hidden to me, but I'm open for suggestions.

**Comment by lvermeulen** [comment]
@cduhard I have released 1.5.0-alpha (fixes a Consul service registration bug) and updated the Nancy net46 sample.
